libs/features/mas/src/inline-price.js

```diff
@@ -92,7 +180,7 @@ export class InlinePrice extends HTMLSpanElement {
             'data-display-annual',
             'data-perpetual',
             'data-promotion-code',
-            'data-tax-exclusive',
+            'data-force-tax-exclusive',
```

Comment:
why change this? if you change this, you need to update inline-price.md as well.

Reply:
Because `data-tax-exclusive` is wrong. We don't use that data attribute. We have `data-force-tax-exclusive` when `exclusive` flag is set to `true`. I will fix inline-price.md.